internal/admin/dashboard/static/css/dashboard.css

```
@@ -0,0 +1,314 @@
+}
+
+.data-table tr:last-child td {
+    border-bottom: none;
+}
+
+.data-table tr:hover td {
+    background: var(--bg-surface-hover);
+}
+
+.mono {
+    font-family: 'SF Mono', 'Menlo', 'Consolas', monospace;
```

Stylelint (17.2.0)

[error] Unexpected quotes around "Menlo" (font-family-name-quotes)

(font-family-name-quotes)